lib/tasks/jenkins.rake

```diff
@@ -1,25 +1,40 @@
 begin
   require "ci/reporter/rake/minitest"
 
-  namespace :foreman do
-    task :set_test_runner do
-      ENV['TESTOPTS'] = "#{ENV['TESTOPTS']} #{Rails.root}/test/test_runner.rb"
+  namespace :jenkins do
+    task :unit do
+      Rails.env = 'test'
+      Rake::Task['db:migrate'].invoke
+      system "rake jenkins:setup:minitest test RAILS_ENV=test"
```

Comment:
Leaving them as they were before would only work if RAILS_ENV=test was specified when calling the task (`rake jenkins:unit RAILS_ENV=test`). 

I tried this and other ways of changing the environment to no avail:
```ruby
+      Rails.env = 'test'
+      Rake::Task['db:test:prepare'].invoke
+      Rake::Task['jenkins:setup:minitest'].invoke
+      Rake::Task['rake:test'].invoke
```

I realize this is a memory hog, but I can't figure out a way of forcing the test environment here without spawning a new Ruby.